modules/claims_api/app/sidekiq/claims_api/v2/disability_compensation_docker_container_upload.rb

```diff
@@ -23,11 +23,12 @@ def perform(claim_id) # rubocop:disable Metrics/MethodLength
                          'Submitting mapped data to Docker container')
 
         evss_res = evss_service.submit(auto_claim, evss_data)
+        raise raise_backend_exception('EVSS500') if evss_res.nil?
```

Comment:
I'm a bit confused on why we added this. did we ever hit this state? on top of that it doesn't work.
```
"undefined method `original_body' for #<NoMethodError: undefined method `raise_backend_exception' for #<ClaimsApi::V2::DisabilityCompensationDockerContainerUpload>>"
```

Reply:
@FonzMP I thought you specifically asked for that line. I will re move it